recipes/libbezier/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "libbezier" %}
+{% set github_org = "dhermes" %}
+{% set github_repo = "bezier" %}
+{% set version = "2020.2.3" %}
+{% set tar_gz_hash = "d2387f9089d2317a80bc97f278843d5326d1bebde7b08c9c9f6caaed9b0a08b1" %}
+{% set build_num = "0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ github_org }}/{{ github_repo }}/archive/{{ version }}.tar.gz
+  sha256: {{ tar_gz_hash }}
+
+build:
+  number: {{ build_num }}
+
+requirements:
+  build:
+    - {{ compiler("fortran") }}
+    - cmake
+
+test:
+  imports:
+
+about:
+  home: https://github.com/dhermes/bezier
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Helper for B&#xe9;zier Curves, Triangles, and Higher Order Objects'
```

Comment:
This help text seems incorrect

Reply:
`&#xe9;` isn't well rendered :wink:

Reply:
Oh haha my bad for being dense. That's an HTML entity for the e with an accent in Bézier. In general I try to keep files ASCII only, but I can just dump the unicode é in there if you think that's the best choice.

Reply:
OK, addressed (and had to learn a bit about YAML + unicode along the way). Thanks for noticing.